app/controllers/application_controller.rb

```diff
@@ -206,8 +206,8 @@ def set_timezone(&block)
   def require_glowfic_domain
     return unless Rails.env.production? || params[:force_domain] # for testability
     return unless standard_request?
-    return if request.host.include?('glowfic.com')
-    return if request.host.include?('glowfic-staging.herokuapp.com')
+    return if request.host.match?(/(^|\.)glowfic\.com$/)
+    return if request.host.match?(/(^|\.)glowfic-staging\.herokuapp\.com$/)
```

Comment:
These should be optional, since `request.host` is not user generated/provided

Reply:
Cool! Which would you prefer, keeping this change or undoing it and marking false positive?